coral/coral/media/js/views/components/workflows/surveys/dc-report-survey.js

```diff
@@ -0,0 +1,339 @@
+define([
+  'knockout',
```

Comment:
Workflows are plugins by the Arches documentation. These workflow files are stored in components/workflows which is wrong. Should be components/plugins